### PR TITLE
Add information about Terraform version and k9s tool for debugging

### PR DESCRIPTION
**Pull request by viniciusdc**
#348

### CODE REVIEW

File: docs/source/06_developers_contrib_guide/04_local_development.md [hunk 1]
```diff
@@ -107,6 +149,40 @@ minikube addons enable metallb
   The 'metallb' addon is enabled
 ```
 
+### Debug your Kubernets cluster 
+
+ K9s is a terminal-based UI to manage Kubernetes clusters that aims to
```
Comment (trallard) [inline docs/source/06_developers_contrib_guide/04_local_development.md:154]:
```suggestion
 [K9s](https://k9scli.io/) is a terminal-based UI to manage Kubernetes clusters that aims to 
```

File: docs/source/06_developers_contrib_guide/04_local_development.md [hunk 2]
```diff
@@ -107,6 +149,40 @@ minikube addons enable metallb
...
 ```
 
+### Debug your Kubernets cluster 
+
+ K9s is a terminal-based UI to manage Kubernetes clusters that aims to 
+ simplify navigating, observing, and managing your applications in K8s. 
+ K9s continuously monitors Kubernetes clusters for changes and provides 
+ shortcut commands to interact with the observed resources becoming a 
+ fast way to review and resolve day-to-day issues in Kubernetes. It's 
+ defiitly a huge improvement to the general workflow and a best-t-have
```
Comment (trallard) [inline docs/source/06_developers_contrib_guide/04_local_development.md:159]:
```suggestion
 definitely a huge improvement to the general workflow and a best-to-have 
```

File: docs/source/06_developers_contrib_guide/04_local_development.md [hunk 3]
```diff
@@ -107,6 +149,40 @@ minikube addons enable metallb
...
+
+---
+**NOTE**
+  In some circumstances you will be confronted with the 
+  necessity to inspect in your ‘localhost’ any services launched by 
+  your cluster. For instance, if your cluster has some problem with the 
+  network traffic tunnel configuration it may limit or block the user's 
+  access to destination resources over the connection. 
+
+K9s port-forward option `<shift-f>` allows you to access and interact
```
Comment (trallard) [inline docs/source/06_developers_contrib_guide/04_local_development.md:179]:
```suggestion
K9s port-forward option `<kbd>shift</kbd> + <kbd>f</kbd>` allows you to access and interact 
```

**Review by laisbsc — COMMENTED** [review]
👍🏽

**Review by laisbsc — APPROVED** [review]

**Pull request merged**